analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -644,4 +661,124 @@ private void writeRepositorySettingsFile(Map<ArtifactRepository, Path> repositor
         return allDependencies;
     }
 
+    private void writeReport(Path rootDir, String filename, String text) throws ManipulationException {
+        final Path reportFile = Paths.get(filename);
+        final Path reportOutputFile;
+
+        if (reportFile.isAbsolute()) {
```

Comment:
We can't default to the project dir ahead of time, I think, so this is what I came up with.

Reply:
By the tie this runs, surely there will be a root project with a build directory?

Reply:
I think the filename is a relative filename, relative to where we will place the file, so we have to resolve it first.

But you mean I am not calling the right API to retrieve the output directory to resolve against?

Reply:
I'm confused?

Reply:
No I mean root directory but then inside `build` surely so repour doesn't commit it?